subprojects/common/file-storage/src/main/kotlin/com/avito/filestorage/RemoteStorageFactory.kt

```diff
@@ -1,26 +1,59 @@
 package com.avito.filestorage
 
 import com.avito.http.HttpClientProvider
-import com.avito.logger.LoggerFactory
-import com.avito.time.TimeProvider
-import okhttp3.HttpUrl.Companion.toHttpUrlOrNull
+import okhttp3.HttpUrl
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.create
+import java.util.concurrent.Executors
 import java.util.concurrent.TimeUnit
 
-object RemoteStorageFactory {
+public object RemoteStorageFactory {
 
-    fun create(
-        endpoint: String,
+    public fun create(
+        endpoint: HttpUrl,
         httpClientProvider: HttpClientProvider,
-        loggerFactory: LoggerFactory,
-        timeProvider: TimeProvider,
+        isAndroidRuntime: Boolean,
```

Comment:
prefer to see two different functions comparing to flag
`createForJvm`
`createForAndroidRuntime`